Tests/SwiftFormatRulesTests/OrderedImportsTests.swift

```diff
@@ -247,6 +247,26 @@ final class OrderedImportsTests: LintOrFormatRuleTestCase {
     XCTAssertDiagnosed(.sortImports)
   }
 
+  func testMultipleCodeBlocksPerLine() {
+    let input =
+      """
+      import A;import Z;import D;import C;
```

Comment:
Can you add a test here that has the imports and the other statements on the same line, just to see how that turns out?

Reply:
Good idea. Done.